exercises/concerned-about-separation/problem/problem.md

```diff
@@ -0,0 +1,52 @@
+This problem is the same as the previous but introduces the concept of **classes**. You will need to create two files to solve this.
```

Comment:
Maybe add a reference to the specific task instead and tell them to complete that first if they haven't already? 

I'm thinking this as there is no specific order tasks have to be done in.